### DISCUSSION TITLE
변수란 무엇인가요?

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#10] [변수]
- 변수란 무엇인가요?

**Comment by pipisebastian** [comment]
>변수란 무엇인가요?

변수란 값을 저장하기 위한 메모리 공간입니다.
변수를 통해 메모리 주소에 직접 접근하지 않고 안전하게 접근할 수 있습니다.

**Comment by publdaze** [comment]
> - 변수란 무엇인가요?
- 값을 저장하기 위해 확보한 메모리 공간 / 메모리 공간 이름
